### ISSUE TITLE
Can it be used locally?

### ISSUE THREAD

**Issue by qcjxberin** [synthiam/Behavior_Control_AimlBot#1]
Can it be used locally?

**Comment by DJSures** [comment]
Yeah - it doesn't require an internet connection. Check out the compiled version here: https://synthiam.com/Support/Skills/Artificial-Intelligence/AimlBot?id=16020
